dist/redhat/cassandra-stress.spec

```diff
@@ -0,0 +1,51 @@
+Name:           cassandra-stress
+Version:        %{version}
+Release:        1
+Summary:        Scylla Cassandra Stress
+Group:          Applications/Databases
+
+License:        Apache
+URL:            http://www.scylladb.com/
+Source0:        cassandra-stress.tar.gz
+BuildArch:      noarch
+Requires:       (jre-11-headless or temurin-11-jre) snappy
```

Comment:
that's a bit limiting, i.e. if one choose different distro of java, it won't be able to install ?

Reply:
Yes it is, but this is what I consider the official supported version of JRE, with which I'm testing locally, there are 3 more which are good candidates: Azul, AWS Correto and Oracle distribution, we can add them if there is a need for it